pkg/kubectl/pdb.go

```diff
@@ -214,7 +214,7 @@ func (s *PodDisruptionBudgetV2Generator) validate() error {
 		return fmt.Errorf("a selector must be specified")
 	}
 	if len(s.MaxUnavailable) > 0 && len(s.MinAvailable) > 0 {
-		return fmt.Errorf("exactly one of min-available and max-available must be specified")
+		return fmt.Errorf("min-available and max-unavailable cannot be both specified")
```

Comment:
@deads2k 
![image](https://user-images.githubusercontent.com/5154334/29248217-3265b694-8045-11e7-8bdd-5b47ddac2abd.png)
from : https://kubernetes.io/docs/tasks/run-application/configure-pdb/
I think it is just string error. =)